site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -57,11 +49,21 @@ This section lists major features that will be added to the Manifest V3 platform
 
 **Tracking issue:** [1247690][]
 
-### Service workers are not started in response to webRequest events
+## Fixed but not yet in Stable {: #resolved }
+
+### webRequest.onAuthRequired events {: #webrequest-onauthrequired }
+
+**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.
```

Comment:
>Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.

This line looks like it needs to be updated. Also, we should write new material to describe that. Who can give me a brain dump about it?

Reply:
@rdcronin implemented this. Here are a few relevant resources. I think I can answer most of the questions you're likely to have about this.

- [Issue 1135492](https://bugs.chromium.org/p/chromium/issues/detail?id=1135492)
- [Design doc](https://docs.google.com/document/d/1iNonK6pgl9U5BTELR3iumfOU6TbirmC--a3GkIjUbfc/edit#heading=h.uoiyzkmb4b5l)
- [Code review](https://crrev.com/c/3923277)

Reply:
I've opened [a ticket](https://github.com/GoogleChrome/developer.chrome.com/issues/4442) so that we can cover this in a separate PR.

Reply:
Resolving this discussion as it's being tracked in #4442.